src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -224,13 +231,15 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
             //cloned opportunity is necessary in order to modify the Primary Contact lookup and update the record in the trigger set
             Opportunity clonedOpp = opp.clone(true,true,false,true);
 
+            Boolean isIndividualAccount = (oppAccountDetails != null && oppAccountDetails.containsKey(opp.AccountId)) ? oppAccountDetails.get(opp.AccountId).npe01__SYSTEMIsIndividual__c : null;
```

Comment:
Removed the null check as an empty Map will be returned from the populateAccountDetails method. 

I decided to leave the code that references the system field since it is the argument that is passed into the getRole and getOCR methods. Its usage makes the code more readable because it clearly shows how the isIndividualAccount variable is populated. I'm open to further discussion.